zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/instance/DbJobState.java

```diff
@@ -427,6 +449,19 @@ public long findBackedOffJobs(final long timestamp, final BiPredicate<Long, JobR
     return nextBackOffDueDate;
   }
 
+  private boolean visit(
```

Comment:
🔧  what's the reason for extraction into a dedicated method? I also think the simple name `visit` is not really a good fit in this case as this is a specific visit logic only relevant for the only calling method `forEachTimedOutEntry` at least `visitTimedOutJob` would be a better fit if you really need to extract it

Reply:
I just thought that the inline method inside `deadlinesColumnFamily.whileTrue` got really longer and unreadable. I will rename the method